OrbitGl/TimeGraph.cpp

```diff
@@ -697,6 +763,41 @@ void TimeGraph::OnRight() {
   NeedsUpdate();
 }
 
+void TimeGraph::SelectAndZoom(const TextBox* text_box) {
+  if (!text_box) return;
+  Zoom(text_box);
+  Select(text_box);
+  NeedsRedraw();
```

Comment:
Is NeedsRedraw() necessary here? After Zoom(...), a NeedsUpdate is needed (and it's called in SetMinMax). 

I guess after Select(...), a redraw is needed, so maybe call NeedsRedraw() in Select(..)?